---
layout: default
---

## About Me

<img class="profile-picture" src="pic.jpg">

Hi! I am David, a student at McGill University graduating in May 2018.  I am starting my PhD in Computer Science in September 2018 and will be choosing a school very soon.  I have been involved in machine learning and bioinformatics research at McGill in addition to other labs all over the world.  I hope to apply my research to data science problems.

For more info see my [GitHub](https://github.com/dvVenuto)

## Research Interest

Convex optimization problems for kernel machines and convolutional neural networks for computer vision.

## Publications

#### Journal Publications

1. **D Venuto**, TD Hocking, L Sphanurattana, M Sugiyama. Support vector comparison machines. Preprint arXiv:1401.8008. *Accepted at JMLR. 2017*. [Link](https://arxiv.org/pdf/1401.8008.pdf)
2. **D Venuto**, G Bourque. Identifying co-opted transposable elements using comparative epigenomics. *Development, Growth and Differentiation*. 2018. [Link](http://onlinelibrary.wiley.com/doi/10.1111/dgd.12423/abstract)
3. C Kanagaratham, V Chiwara, B Ho, S Moussette, M Youssef, **D Venuto**, L Jeannotte, G Bourque, JB de Sanctis, D Radzioch, A Naumova. Loss of the zona pellucida binding protein 2 (Zpbp2) gene in mice impacts airway hypersensitivity and lung lipid metabolism in a sex-dependent fashion. *Accepted to Mammalian Genome*. 2018.

#### Conference Presentations

1. **D Venuto**, et al., Comparative epigenomics in primate immune cells reveals putatively exapted transposable element families. *FASEB Meeting 2017*, Big Sky. (Short Talk)

2. **D Venuto**, et al., Mapping functional non-coding transcripts in primate immune cells using compara- tive epigenomics. *IHEC* 2016, Brussels. (Poster)

## News

- My novel learning to rank method is available on arXiv and is now under review at JMLR.
- I recently had my comparative epigenomics paper accepted.


